lib/ui/map/map_page/map_page.dart

```diff
@@ -420,6 +422,7 @@ class _MapPageState extends PageStatePlante<MapPage>
                     child: AnimatedListSimplePlante(children: _fabs()))),
             loadShopsButton,
             MapBottomHint(_bottomHint),
+            _progressBar(),
```

Comment:
Wouldn't that put the progress bar above of buttons "Done" and "Cancel" in the modes of the map, which have the buttons?
Maybe it should be below `AnimatedListSimplePlante(children: _mode.buildBottomActions())`?

Reply:
ah probably, let me see check that